subsys/net/lib/nrf_cloud/Kconfig.nrf_cloud_coap

```diff
@@ -37,6 +37,15 @@ config NON_RESP_RETRIES
 	  result in up to this number of retransmissions of the request followed
 	  by waits for a response.
 
+config NRF_CLOUD_COAP_GF_CONF
```

Comment:
This is not used anywhere at the moment but I assume it would be in `coap_codec.c`

Reply:
Premature push.

Reply:
Still cannot find this used anywhere...